CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+CHANGELOG
+=========
+
+## HEAD (Unreleased)
+
+## 0.0.8 (2020-12-03)
+
+- Use ephemeral storage for disk mutations.
+  [#109](https://github.com/pulumi/pulumi/pull/109)
+
+- Fix handling of `state` message, and add new `lastAttemptedCommit` and `lastSuccessfulCommit` status fields.
+  [#107](https://github.com/pulumi/pulumi/pull/107).
+
+- Support configuring alternative `backend`s.
+  [#106](https://github.com/pulumi/pulumi/pull/106)
+
+- Support streaming logs from `pulumi up/destroy/refresh`.
+  [#101](https://github.com/pulumi/pulumi/pull/101)
+
+## 0.0.7 (2020-09-29)
+
+- Add SSH keyagent, add keys to known_hosts for SSH git.
+  [#92](https://github.com/pulumi/pulumi/pull/92)
+
+## 0.0.6 (2020-09-15)
+
+- Refactor controller to use Automation API.
+  [#86](https://github.com/pulumi/pulumi/pull/86)
```

Comment:
`s/pulumi/pulumi-kubernetes-operator` for all repo link refs in this file

Reply:
Hah nice catch @elsesiy :). @lukehoban let me know if you need me to make the change.